rpcs3/rpcs3qt/vfs_dialog_path_widget.cpp

```diff
@@ -85,13 +81,21 @@ vfs_dialog_path_widget::vfs_dialog_path_widget(const QString& name, const QStrin
 
 	connect(m_dir_list, &QListWidget::itemDoubleClicked, this, [](QListWidgetItem* item)
 	{
-		if (!item) return;
```

Comment:
Removing this line makes no sense. There is no guarantee that this is always valid.

Reply:
It's best practise. you don't just dereference pointers from a thirdparty library

Reply:
ok done